neuraxle/hyperparams/distributions.py

```diff
@@ -1007,3 +1016,141 @@ def narrow_space_from_best_guess(self, best_guess, kept_space_ratio: float = 0.5
         return Normal(
             new_mean, new_std, self.hard_clip_min, self.hard_clip_max, self.null_default_value
         ).was_narrowed_from(kept_space_ratio, self)
+
+
+class DistributionMixture(HyperparameterDistribution):
+    """Get a mixture of multiple distribution"""
+
+    def __init__(self, distributions: Union[List[HyperparameterDistribution], Tuple[HyperparameterDistribution, ...]],
+                 distribution_amplitudes: Union[List[float], Tuple[float, ...]]):
+        """
+        Create a mixture of multiple distributions.
+
+        :param distributions: list of multiple instantiated distribution.
+        :param distribution_amplitudes: list of float representing the amplitude for each distribution.
+        """
+        # Normalize distribution amplitude
+        distribution_amplitudes = np.array(distribution_amplitudes)
+        distribution_amplitudes = distribution_amplitudes / np.sum(distribution_amplitudes)
+        self.distributions = distributions
+        self.distribution_amplitudes = distribution_amplitudes
+
+    @staticmethod
+    def build_gaussian_mixture(distribution_amplitudes: Union[List[float], Tuple[float, ...]],
+                               means: Union[List[float], Tuple[float, ...]],
+                               stds: Union[List[float], Tuple[float, ...]],
+                               distributions_mins: Union[List[float], Tuple[float, ...]],
+                               distributions_max: Union[List[float], Tuple[float, ...]], use_logs: bool = False, use_quantized_distributions: bool = False):
+
+        distribution_class = Normal
+
+        if use_logs:
+            distribution_class = LogNormal
+
+        distributions = []
+
+        for mean, std, single_min, single_max in zip(means, stds, distributions_mins, distributions_max):
+
+            if np.isneginf(single_min):
+                single_min = None
+
+            if np.isposinf(single_max):
+                single_max = None
+
+            distribution_instance = distribution_class(mean, std, hard_clip_min=single_min, hard_clip_max=single_max)
+
+            if use_quantized_distributions:
+                distribution_instance = Quantized(distribution_instance)
```

Comment:
Note that in the future, I'm planning to add arguments to Quantized distributions to make them not only perfect integers, but rather, also add an interval such as `interval=0.1` to have more precision on the quantification, and perhaps even a fixed phase to start the quantization, such as saying also `anchor=0.05`. 

So a `Quantized(Uniform(0, 1), interval=0.1, anchor=0.05)` would generate something like `[0.05, 0.15, 0.25, ..., 0.95]` as it is anchored to 0.05.

This said, the default values for those new hyperparams would for sure be `interval=1, anchor=0`. I'm still not sure if we'll use integers or floats, as it might be both. I suggest that it be a float, and that if we want integers, that we need to do something with another wrapper such as `Round(Quantized(Uniform(0, 10), interval=2)))` for instance to force rounding.

I just created issue #225 for this review comment (you can resolve, issue is already created to do it in another PR later). 

For now, the only thing is that in consideration of those future changes, I don't know what to do here with what you added, as it will have more optional arguments in the future: 
- `distribution_instance = Quantized(distribution_instance)`. 

It might be okay, just sharing thoughts on future improvements.

Reply:
Will modify code when the new argument are added to the quantized distribution.